src/libs/layout/header/index.tsx

```diff
@@ -59,6 +66,11 @@ export function PopupHeader({
   );
   const activeAccount = useSelector(selectVaultActiveAccount);
 
+  useActiveAccountBalance();
+  useFetchAccountActivity(ActivityListTransactionsType.All);
+  useNftTokens();
+  useErc20Tokens();
```

Comment:
<img width="1027" alt="image" src="https://github.com/make-software/casper-wallet/assets/44294945/76d4330f-e1a0-477e-8152-74213540e61b">

We have issues in `useFetchAccountActivity` and `useErc20Tokens`